### PR TITLE
Fix date offset problem when showing vitals

### PR DESCRIPTION
**Pull request by edrisse**

### CODE REVIEW

**Comment by edrisse** [comment]
[Let back-end determine current date](https://trello.com/c/ZCKAhOtb/66-let-back-end-determine-current-date)

File: app/common/ui-helper/filters/momentFilter.js [hunk 1]
```diff
@@ -13,14 +13,18 @@
         .filter("moment", () => filter);
 
     function filter(input, format) {
-        if (moment.isMoment(input)) {
+        var momentInput = input;
+        if (typeof input === 'string') {
```
Comment (ynurmahomed) [inline app/common/ui-helper/filters/momentFilter.js]: I think its possible to always wrap the input inside `moment` regardless of the type, eliminating the need for `if(moment.isMoment(...))`.

File: app/common/ui-helper/filters/momentFilter.js [hunk 2]
```diff
@@ -13,14 +13,18 @@
...
+            momentInput = moment(input);
+        }
+        if (moment.isMoment(momentInput)) {
             var customFormat = customFormats[format];
             if (customFormat) {
                 format = customFormat;
             } else if (!format) {
                 format = 'DD/MM/YYYY';
```
Comment (ynurmahomed) [inline app/common/ui-helper/filters/momentFilter.js]: Can we use the same `dateFormat` from `application/constants.js`?

File: app/common/ui-helper/filters/momentFilter.js (hunk 1 above)
Comment (edrisse) [reply app/common/ui-helper/filters/momentFilter.js]: We need the if so we don't try to process anything other than `string` and `moment` objects.

**Comment by edrisse** [comment]
Changes made. Please review again.

**Pull request merged**